app/components/support_interface/duplicate_matches_table_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Match</th>
+      <th scope="col" class="govuk-table__header">Date</th>
+    </tr>
+  </thead>
+
+  <tbody class="govuk-table__body">
+    <% @matches.each do |match| %>
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell">
+          <%= govuk_link_to SupportInterface::DuplicateMatchesTableComponent.description_for(match), support_interface_duplicate_match_path(match) %>
```

Comment:
I think elsewhere we make helpers like this instance methods and call without the qualifying class?

Reply:
ah — I see, it's called out of context below. carry on!